README.md

```diff
@@ -49,6 +59,57 @@ Then:
+* `use_black_formatter` - Use [black](https://github.com/psf/black) for formatting your code (default: n)
+* `use_spellcheck` - Use [spellchecking](https://github.com/rfk/pyenchant) and user configurable dictionary in `.pylintrc` (default: n)
+* `use_tox` - EXPERIMENTAL: Adds support for tox (default: n)
+* `use_makefile` - EXPERIMENTAL: Adds support for Makefiles (default: n)
+* `ci` - Select the CI system you want to use. By default the CI system is chosen based on the selected versioning system (`vs`). If you chose Github as VS the default CI system will be Azure Pipelines.  In case of Bitbucket you will get Jenkins.
+* `ci_url` - URL to your CI system (eg. https://dev.azure.com, https://jenkins.acme.org)
+* `ci_org_name` - For Azure Pipelines this is the organization name. In case of Jenkins you can leave this either empty or in case you have defined your Jenkins projects under a folder use the folder name.
+* `ci_project_name` - For Azure Pipelines this is the name of your project. In case of Jenkins this is by default `vs_account` which usually is a `BitBucket Team Project`
+* `pypi_repo` - PyPI repository the package should be deployed to. In case you select Github for `vs` the default value is the official PyPI index whereas in case of BitBucket you get a hint to provide a URL to an Artifactory PyPI repository.
+* `pypi_credentials_id` - For Azure Pipelines this is the [service connection](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml) that allows you to push Python packages to the official PyPI index. For Jenkins this is the credentials ID used for authentication with an Artifactory PyPI repository.
+* `docker_registry_credentials_id` - For Azure Pipelines this is the service connection that allows you to push Docker images to a public Docker registry. For Jenkins this is the credentials ID used for authentication with for example an Artifactory Docker repository.
+* `ci_ssh_agent_user` -  Skip this for Azure Pipelines. For Jenkins this is the credentials ID for a user authenticated with a SSH private key. This is required to allow git commits during the versioning process.
+* `azure_build_definition_id` - Skip this for Jenkins. For Azure Pipelines this is the ID identifiying the pipeline (eg. https://dev.azure.com/tomtomweb/GitHub-TomTom-International/_build?definitionId=26)
+
+## Tips
+
+In case you want to host a project in the `tomtom-international` organization and you chose Azure for the CI contact the @nav-pipeline team to request a build pipeline in the [TomTom Azure DevOps account](https://dev.azure.com/tomtomweb/GitHub-TomTom-International/_build) (@nav-pipeline: detailed instructions on how to set up a project can be found [here](https://github.com/tomtom-international/azure-pipeline-templates/blob/master/README.md)).
+
+Contact as well the @nav-pipeline team to activate your project on `pyup.io` for automated scanning of your Python dependencies.
```

Comment:
Same as earlier comment do we want to keep this still? If yes then also worth to group together with the other part.